Homework7/Item.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Homework7
+{
+    public abstract class Item 
+    {
+        public string? modelName;
+        public decimal price;
+        string? Description { get; }
+        public Item(string modelName, decimal price) 
+        {
+            this.modelName = modelName;
+            this.price = price;
+        }
+        public void TurnOn()
```

Comment:
It should be abstract, then class ItemWorkingWithPaper also will be abstract and TurnOn method as well. Implementation will be done in derived classes

Reply:
I have made Turn On() abstract, removed its body. 
Also child methods are overridden and ItemWorkingWithPaper class was changed to abstract now.